gateway/src/main/scala/com/improving/app/gateway/infrastructure/routes/DemoScenarioGatewayRoutes.scala

```diff
@@ -51,221 +50,165 @@ import scala.concurrent.Future
-                        .shuffle(firstNames)
-                        .take(numMembers)
-                        .zip(Random.shuffle(lastNames).take(numMembers))
-                    )
-                    .map { case ((id, (creatingMember, tenant)), (firstName, lastName)) =>
-                      memberHandler
-                        .registerMember(
-                          RegisterMember(
-                            id,
-                            Some(
-                              EditableMemberInfo(
-                                handle = Some(s"${firstName.take(1)}.${lastName.take(3)}"),
-                                avatarUrl = Some(s"${org.organizationInfo.get.shortName}.org"),
-                                firstName = Some(firstName),
-                                lastName = Some(lastName),
-                                notificationPreference =
-                                  Some(NotificationPreference.NOTIFICATION_PREFERENCE_APPLICATION),
-                                contact = Some(
-                                  EditableContact(
-                                    firstName = Some(firstName),
-                                    lastName = Some(lastName),
-                                    emailAddress = Some(
-                                      s"${firstName.toLowerCase}.${lastName.toLowerCase}@${org.organizationInfo.get.shortName}.org"
-                                    ),
-                                    phone = Some(genPhoneNumber),
-                                    userName = Some(s"${firstName.take(1)}$lastName")
-                                  )
-                                ),
-                                organizationMembership = Seq(org.getOrganizationId),
-                                tenant = tenant
-                              )
-                            ),
-                            creatingMember
-                          )
-                        )
-                        .flatMap { memberRegistered: MemberRegistered =>
-                          memberHandler
-                            .activateMember(
-                              ActivateMember(memberRegistered.memberId, memberRegistered.meta.map(_.getCreatedBy))
-                            )
-                            .map(_ => memberRegistered.toMember)
-                        }
-                    }
-                )).flatten
-                .map(_.map { member =>
-                  logger.debug(s"Member successfully established with $member")
-                  member
-                })
-
-              complete {
-                for {
-                  org <- orgResponse
-                  tenants <- tenantResponses
-                  members <- memberResponses
-                } yield ScenarioStarted(tenants = tenants, organizations = Seq(org), members = members).toProtoString
-              }
-            }
-          }
-        }
-      }
-    }
+  // def demoScenarioRoutes(
```

Comment:
commented out code

Reply:
The is a TODO on top saying when to take this out